config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "PORT": 8080,
+  "NODE_ENV": "development",
+  "BASE_PATH": "/api"
+}
```

Comment:
new line please

Reply:
Anyway, I've added the eslint json plugin to also validate json files.